internal/install/recipe_installer.go

```diff
@@ -169,24 +170,27 @@ Welcome to New Relic. Let's set up full stack observability for your environment
 		"RecipeNamesProvided": i.RecipeNamesProvided(),
 	}).Debug("context summary")
 
+	i.status.InstallStarted()
+
+	if i.RecipeNamesProvided() {
```

Comment:
I believe this SetTargetedInstall( ) should be done before the InstallStarted( ) otherwise the Started event is going to miss the targeted info.

Reply:
Didn't realize that, will update.